### ISSUE TITLE
Support only tags for modules in the registry, any SHA in the proxy

### ISSUE THREAD

**Issue by arschles** [gomods/athens#301]
This was discussed in the July 12, 2018 meeting, and I think it's a good idea because:

- Folks should be able to choose to serve whatever code they want in their proxies - especially for internal (i.e. "behind the firewall") proxy deployments
- We want to set the standard that every module in the global registry is an official, well defined release, and a good way to establish that expectation is that each module is tagged (for some definition of 'tag') in the given VCS

**Comment by marpio** [comment]
@arschles Do you think we still need this one? #699 is on hold (but should be fixed at some point) and we removed registry.

**Comment by robjloranger** [comment]
I agree this should be no longer needed once #699 is done.

**Comment by arschles** [comment]
Agree with you @marpio (1.5 years later 😆 ) - closing
